### PR TITLE
Some minor Changes

### PR DESCRIPTION
**Pull request by BaochengSu**
1. kconfig: Improve the hit for the debian mirror URL, `http://` protocol prefix is necessary.

2. Link generated .wic to .wic.img for better backward compatibility.

### CODE REVIEW

File: recipes-core/images/iot2050-image-base.bb [hunk 1]
```diff
@@ -56,3 +56,10 @@ python image_postprocess_restore_sources_list () {
     else:
         bb.note('No need to restore sources for mainline packages')
 }
+
+# Make the .wic.img symlink to the .wic file for better backward compatibility
+do_deploy() {
+    echo "Linking wic img"
+    rm -f ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic.img
+    ln -s ${IMAGE_FULLNAME}.wic ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic.img
+}
```
Comment (jan-kiszka) [inline recipes-core/images/iot2050-image-base.bb:65]: Let's also update the README to the new name.

File: recipes-core/images/iot2050-image-base.bb [hunk 2]
```diff
@@ -56,3 +56,10 @@ python image_postprocess_restore_sources_list () {
     else:
         bb.note('No need to restore sources for mainline packages')
 }
+
+# Make the .wic.img symlink to the .wic file for better backward compatibility
+do_deploy() {
+    echo "Linking wic img"
+    rm -f ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic.img
+    ln -s ${IMAGE_FULLNAME}.wic ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic.img
```
Comment (AsuraZeng) [inline recipes-core/images/iot2050-image-base.bb:64]: Should the full path be used here for .wic

Comment (BaochengSu) [reply recipes-core/images/iot2050-image-base.bb:64]: No, the full path could not be determined, it is different across the build env and the host env, neither across different host env.

Comment (AsuraZeng) [reply recipes-core/images/iot2050-image-base.bb:64]: I mean 
```
ln -s ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic.img
```

Comment (BaochengSu) [reply recipes-core/images/iot2050-image-base.bb:64]: > I mean
> 
> ```
> ln -s ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic.img
> ```

The `${DEPLOY_DIR_IMAGE}` is not the correct host path if the image is built with docker.

Comment (jan-kiszka) [inline recipes-core/images/iot2050-image-base.bb:64]: `ln -fs`?

File: recipes-core/images/iot2050-image-base.bb [hunk 3]
```diff
@@ -56,3 +56,10 @@ python image_postprocess_restore_sources_list () {
     else:
         bb.note('No need to restore sources for mainline packages')
 }
+
+# Make the .wic.img symlink to the .wic file for better backward compatibility
+do_deploy() {
+    echo "Linking wic img"
+    rm -f ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic.img
+    ln -sf ${IMAGE_FULLNAME}.wic ${DEPLOY_DIR_IMAGE}/${IMAGE_FULLNAME}.wic.img
```
Comment (jan-kiszka) [inline recipes-core/images/iot2050-image-base.bb:64]: No, I (also) meant that the `ln -f` would obsolete the `rm`. Or are there other reasons to have that?

**Review by jan-kiszka — APPROVED** [review]

**Pull request merged**